contentcuration/contentcuration/frontend/channelEdit/store.js

```diff
@@ -1,16 +1,32 @@
 import template from './vuex/template';
 import assessmentItem from './vuex/assessmentItem';
+import clipboard from './vuex/clipboard';
 import contentNode from './vuex/contentNode';
 import currentChannel from './vuex/currentChannel';
 import importFromChannels from './vuex/importFromChannels';
 import task from './vuex/task';
+import * as actions from './actions';
+import * as getters from './getters';
+import * as mutations from './mutations';
 import storeFactory from 'shared/vuex/baseStore';
+import persistFactory from 'shared/vuex/persistFactory';
 
 const store = storeFactory({
+  state() {
+    return {
+      viewMode: null,
```

Comment:
Could you add comments on what the difference between viewMode and viewModeOverrides is?